admin-dev/themes/default/js/jquery.fileupload-validate.js

```diff
@@ -9,48 +9,45 @@
  * http://www.opensource.org/licenses/MIT
  */
 
-/*jslint nomen: true, unparam: true, regexp: true */
-/*global define, window */
+/* jslint nomen: true, unparam: true, regexp: true */
```

Comment:
Woot, just notice about jslint :scream: 
I think you can exclude these dependencies, no?

Reply:
Or remove dependencies and add it in package.json ?

Reply:
I'll remove it

Reply:
Oh, I just noticed it's a lib, we shouldn't modify anything to this file!